api/src/main/java/com/budgettracker/api/models/UserCategory.java

```diff
@@ -0,0 +1,35 @@
+package com.budgettracker.api.models;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+import java.math.BigInteger;
+import java.time.LocalDateTime;
+
+@Getter
+@Setter
+@Entity(name="userCategory")
+@Table(name="userCategories")
```

Comment:
i think it is ok the table name, but would be better as "user_categories", easier to find in db